utils/build.ps1

```diff
@@ -1438,12 +1438,7 @@ function Build-CMakeProject {
           }
 
           Add-KeyValueIfNew $Defines CMAKE_Swift_COMPILER $SWIFTC
-          if ($UsePinnedCompilers.Contains("Swift")) {
-            # TODO(compnerd): remove this once we bump to a newer pinned toolchain.
-            Add-KeyValueIfNew $Defines CMAKE_Swift_COMPILER_TARGET $Platform.Triple
-          } else {
-            Add-KeyValueIfNew $Defines CMAKE_Swift_COMPILER_TARGET (Get-ModuleTriple $Platform)
-          }
+          Add-KeyValueIfNew $Defines CMAKE_Swift_COMPILER_TARGET (Get-ModuleTriple $Platform)
```

Comment:
This seems unsafe - if the older compiler doesn't support the feature, this will break.

Reply:
We have updated the bootstrap toolchain, this does not seem necessary anymore.

Reply:
The bootstrap toolchain is 6.1.2, is that new enough?